test/session.js

```diff
@@ -74,7 +76,7 @@ tap.test('Create session', withServer((t, server) => {
     })
   })
 
-  t.test('should return 201 status when a session is created', withUser((t, user) => {
+  t.test('should return 201 status when a session is created with backward compatible jwt config', withUser((t, user) => {
```

Comment:
Same as a previous comment, im wondering if we should explicitly set the deprecated `secret` property to illiterate that it still works. At the moment, its still within the default config which seems to show its still current/accepted property to use